static/main.css

```
@@ -992,6 +1027,60 @@ footer .logo {
   opacity: 1;
 }
 
+@media (min-width: 1000px) {
+  .menu-collapsed {
+    #menu-content {
+      .menu-group {
+        display: none;
+      }
```

WDYT about
```suggestion
      .menu-group {
        margin-bottom: 0;
        span {
          display: none;
        }
      }
```

I kind of missed the grouping when the menu is collapsed.